mantis-control-plane/mantis-control-plane-server/src/test/java/io/mantisrx/master/api/akka/route/JacksonTest.java

```diff
@@ -30,53 +31,45 @@
 import java.io.ObjectOutputStream;
 import java.util.List;
 import java.util.Optional;
+
+import io.mantisrx.shaded.com.fasterxml.jackson.databind.SerializationFeature;
+import io.mantisrx.shaded.com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
 import org.junit.Test;
 
 
 public class JacksonTest {
-    private static final ObjectMapper objectMapper = new ObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+    private static final ObjectMapper objectMapper = new ObjectMapper()
+        .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+        .configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false)
+        .registerModule(new Jdk8Module());
```

Comment:
As far as I know this introduced features in Jackson like not requiring `@JsonProperty` in Java 8. As best I can tell these are now all handled via Jackson. We can likely skip the module, but probably no harm in including it.

Reply:
iirc these were added due to some datetime format test failure.